tools/format/examples/vehicle_part.json

```diff
@@ -14,14 +14,15 @@
     "breaks_into": [
       { "item": "steel_lump", "count": [ 3, 6 ] },
       { "item": "steel_chunk", "count": [ 3, 6 ] },
-      { "item": "scrap", "count": [ 3, 6 ] }
+      { "item": "scrap", "min": 3, "max": 6 }
```

Comment:
> If `min`, `max` and `count` are interchangeable we should perhaps aim to support only one variant?

Yeah, that sounds good.

> It looks like `min` and `max` aren't actually read.

Good catch. It looks like you beat me to it, thanks.